systems/controllers/cp_traj_gen.cc

```diff
@@ -266,8 +270,8 @@ PiecewisePolynomial<double> CPTrajGenerator::createSplineForSwingFoot(
   Y[2](1, 0) = CP(1);
   // z
   Y[0](2, 0) = init_swing_foot_pos(2);
-  Y[1](2, 0) = mid_foot_height_;
-  Y[2](2, 0) = desired_final_foot_height_;
+  Y[1](2, 0) = mid_foot_height_ + stance_foot_height(0);
```

Comment:
Can you add a comment about why we're adding the z-component of the stance foot to the spline?

Reply:
done